release.doap

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Project
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+    xmlns:foaf="http://xmlns.com/foaf/0.1/"
+    xmlns:gnome="http://api.gnome.org/doap-extensions#"
+    xmlns="http://usefulinc.com/ns/doap#">
+    
+  <name xml:lang="en">gtk-rs</name>
+  <shortdesc xml:lang="en">Rust bindings for glib, GTK, and other GNOME libraries.</shortdesc>
+  <description>The gtk-rs project provides safe bindings to the Rust language for fundamental libraries from the GNOME stack like GLib, Cairo, GTK 3 and GTK 4.</description>
```

Comment:
```suggestion
  <description>The gtk-rs project provides safe bindings to the Rust language for fundamental libraries from the GNOME stack like GLib, Cairo and GTK 4.</description>
```

GTK 3 bindings are not maintained anymore since almost 2 years so probably not good to mention them here

Reply:
This was just copy-paste from the page. Opened an issue there as well https://github.com/gtk-rs/gtk-rs.github.io/issues/313